### PR TITLE
Update and rename README to README.md

### PR DESCRIPTION
**Pull request by nsesodia**
Formatted README for better readability and scanability using markup. Added section about using octave software for users (not developers).

### CODE REVIEW

**Comment by siko1056** [comment]
Thank for the pull-request (PR) @nsesodia .

This GitHub repository is just a mirror of the mercurial repository, thus there is no guarantee that Octave developers will review this PR.  Maybe @mmuetzel can comment how to go forward with changes here.

octave_github

About the changes themselves.  The GNU Standards of `autotool` (that Octave is using) demand a file to be called `README` without extension. https://www.gnu.org/software/automake/manual/html_node/Strictness.html

The Markdown changes look good, but it depends what the Octave developers prefer, as only pages like GitLab/GitHub display the `README` file nicely and so far the Octave project decided not to fully embrace GitLab/GitHub.

**Comment by cbm755** [comment]
Perhaps we can make the formatting changes but leave the file as `README` (not `README.md`?)

File: README.md [hunk 1]
```diff
@@ -29,51 +35,48 @@ WITHOUT ANY WARRANTY; without even the implied warranty of
 MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 GNU General Public License for more details.
 
-You should have received a copy of the GNU General Public License
-along with Octave; see the file COPYING.  If not, see
-<https://www.gnu.org/licenses/>.
+You should have received a copy of the 
+[`GNU General Public License`](https://github.com/gnu-octave/octave/blob/default/COPYING)
```
Comment (mmuetzel) [inline README.md:39]: The GitHub mirror is just that (a mirror of the main repository).
We shouldn't hard-code paths to the mirror repository here.

File: README.md [hunk 2]
```diff
@@ -29,51 +35,48 @@ WITHOUT ANY WARRANTY; without even the implied warranty of
...
+Octave requires approximately `475 MB` of disk storage to unpack and
+compile from source (significantly more, 3.8 GB, if you compile with
+debugging symbols).  Once installed, Octave requires approximately
+`75 MB` of disk space (again, considerably more, 415 MB, if you don't
+build shared libraries or the binaries and libraries include
+debugging symbols).
 
-Bugs and Patches
-----------------
+See [Octave installation guide](https://github.com/gnu-octave/octave/blob/default/doc/interpreter/install.txi)
```
Comment (mmuetzel) [inline README.md:72]: `.txi` files are sources from which documentation is generated. Users shouldn't need to read those sources.

Maybe, use a link to the generated HTML documentation?
https://docs.octave.org/latest/Installation.html

File: README.md [hunk 3]
```diff
@@ -97,6 +100,12 @@ always welcome.  Reports with patches are even more welcome.
...
 
-Up to date information about Octave is available on the WWW at
+Up to date information about Octave is available at
 <https://www.octave.org>, or ask for help via email
 <help@octave.org>.
+
+Copyright (C) 1996-2024 The Octave Project Developers
+
+See the file 
+[`COPYRIGHT.md`](https://github.com/gnu-octave/octave/blob/default/COPYRIGHT.md)
```
Comment (mmuetzel) [inline README.md:110]: Same as above:

The GitHub mirror is just that (a mirror of the main repository).
We shouldn't hard-code paths to the mirror repository here.

**Comment by mmuetzel** [comment]
Some of the rewording of the file is probably ok.
I'm uncertain if changing to markdown is compliant to the autotools standard though.

File: README.md [hunk 4]
```diff
@@ -29,51 +35,48 @@ WITHOUT ANY WARRANTY; without even the implied warranty of
...
-<https://www.octave.org/download.html>.  The current development
-sources may be found under the Source Code tab on
-[Savannah](https://savannah.gnu.org/projects/octave/).
-
-Installation
-------------
+[here](https://ftp.gnu.org/gnu/octave)
+and many mirror sites around the world.  You may also find links to binary distributions
+[here](https://www.octave.org/download.html).
+The current development sources may be found under the Source Code tab on [Savannah](https://savannah.gnu.org/projects/octave/).
```
Comment (mmuetzel) [inline README.md:49]: Try to limit line length to 80 characters.

**Comment by mmuetzel** [comment]
No more reaction of the original author in more than 2 weeks. Closing PR.
It can be reopened when there is further activity.

**Pull request closed without merging**